src/Test/contquadratic.jl

```diff
@@ -50,20 +50,29 @@ function qp1test(model::MOI.ModelLike, config::TestConfig)
 
         @test MOI.get(model, MOI.ObjectiveValue()) ≈ 13/7 atol=atol rtol=rtol
 
-        @test MOI.get(model, MOI.VariablePrimal(), v) ≈ [4/7,3/7,6/7] atol=atol rtol=rtol
+        @test MOI.get(model, MOI.VariablePrimal(), v) ≈ [4/7, 3/7, 6/7] atol=atol rtol=rtol
+
+        if config.duals
+            @test MOI.get(model, MOI.DualStatus()) == MOI.FEASIBLE_POINT
+            # The dual constraint gives
+            # [λ_c1 + λ_c2, 2λ_c1 + λ_c2, 3λ_c1] = [2 1 0; 1 2 1; 0 1 2] * x
+            #                                    = [11, 16, 15] / 7
+            # hence λ_c1 = 5/7 and λ_c2 = 6/7.
+            @test MOI.get(model, MOI.ConstraintDual(), c1) ≈ 5/7 atol=atol rtol=rtol
```

Comment:
I haven't checked these, but we'll find out soon enough if the solvers agree.

Reply:
ECOS agrees but he currently can't do quadratic objectives until we have objective bridge so it only did socp's